shelf_router/lib/src/router.dart

```diff
@@ -84,6 +95,11 @@ class Router {
     }
     verb = verb.toUpperCase();
 
+    if (verb == 'GET') {
```

Comment:
What happens if you specify both `HEAD` and `GET`?

Reply:
As long as you specify the `HEAD` handler before the `GET` handler you'll be fine.

Reply:
This should be documented somewhere, otherwise this is good to go.

Reply:
Good point, I documented it for `Router.get` but we should also document it here.